python/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-alpine as resource
+RUN apk add --update \
+    git \
+    zip \
+    unzip \
+    && rm -rf /var/cache/apk/*
```

Comment:
As of Alpine Linux 3.3 there exists a new --no-cache option for apk. It allows users to install packages with an index that is updated and used on-the-fly and not cached locally: